cognite_toolkit/_cdf_tk/storageio/_raw.py

```diff
@@ -81,3 +88,9 @@ def configurations(self, selector: RawTableSelector) -> Iterable[StorageIOConfig
         yield StorageIOConfig(
             kind=RawTableCRUD.kind, folder_name=RawTableCRUD.folder_name, value=selector.table.model_dump(by_alias=True)
         )
+
+    def row_to_resource(self, row: dict[str, JsonVal], selector: RawTableSelector | None = None) -> RowWrite:
+        key = str(uuid4())
+        if selector is not None and selector.key is not None and selector.key in row:
+            key = str(row.pop(selector.key))
+        return RowWrite(key=key, columns=row)
```

Comment:
Implemented for raw with option for selecting which column to use as key, thus solving the feature request: CDF-24132

Reply:
Minor thing, but should we at least print a warning if the selected key is not present in the Manifest file?